traffic_portal/app/src/common/modules/form/serviceCategory/form.serviceCategory.tpl.html

```diff
@@ -0,0 +1,58 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div class="x_panel">
+    <div class="x_title">
+        <ol class="breadcrumb pull-left">
+            <li><a ng-click="navigateToPath('/servicecategories')">Service Categories</a></li>
+            <li class="active">{{serviceCategoryName}}</li>
+        </ol>
+        <div class="pull-right" role="group" ng-show="!settings.isNew">
+            <button class="btn btn-primary" title="View Delivery Services" ng-click="viewDSs()">View Delivery Services</button>
+        </div>
+        <div class="clearfix"></div>
+    </div>
+    <div class="x_content">
+        <br>
+        <form name="serviceCategoryForm" class="form-horizontal form-label-left" novalidate>
+            <div class="form-group" ng-class="{'has-error': hasError(serviceCategoryForm.name), 'has-feedback': hasError(serviceCategoryForm.name)}">
+                <label for="name" class="control-label col-md-2 col-sm-2 col-xs-12">Category Name *</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <input id="name" name="name" type="text" class="form-control" ng-model="serviceCategory.name" required autofocus>
+                    <small class="input-error" ng-show="hasPropertyError(serviceCategoryForm.name, 'required')">Required</small>
+                    <span ng-show="hasError(serviceCategoryForm.name)" class="form-control-feedback"><i class="fa fa-times"></i></span>
+                </div>
+            </div>
+            <div class="form-group" ng-class="{'has-error': hasError(serviceCategoryForm.tenantId), 'has-feedback': hasError(serviceCategoryForm.tenantId)}">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Tenant</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <select name="tenantId" class="form-control" ng-model="serviceCategory.tenantId" ng-options="tenant.id as tenantLabel(tenant) for tenant in tenants" required>
+                        <option value="">Select...</option>
```

Comment:
this is where you could actually use that other syntax because this field is required so once something is selected it's ok for the Select... to disappear forever.

`<option hidden disabled selected value="">Select...</option>`